stellar-dotnet-sdk/requests/PaymentsRequestBuilder.cs

```diff
@@ -1,52 +1,92 @@
-﻿using stellar_dotnet_sdk.responses;
-using stellar_dotnet_sdk.responses.operations;
-using stellar_dotnet_sdk.responses.page;
+﻿using stellar_dotnet_sdk.responses.operations;
 using System;
 using System.Net.Http;
-using System.Threading.Tasks;
 
-namespace stellar_dotnet_sdk.requests
+namespace stellar_dotnet_sdk.requests;
+
+public interface IPaymentsRequestInitialBuilder : IPaymentsRequestBuilder
+{
+    IPaymentsRequestBuilder ForAccount(string account);
+    IPaymentsRequestBuilder ForLedger(long ledgerSeq);
+    IPaymentsRequestBuilder ForTransaction(string transactionId);
+}
+
+public interface IPaymentsRequestBuilder : IRequestBuilderStreamable<OperationResponse>, IRequestBuilderExecutePageable<IPaymentsRequestBuilder, OperationResponse>
 {
-    public class PaymentsRequestBuilder : RequestBuilderStreamable<PaymentsRequestBuilder, OperationResponse>
+    IPaymentsRequestBuilder JoinTransactions(bool value);
+}
+
+public class PaymentsRequestBuilder : RequestBuilderStreamable<IPaymentsRequestBuilder, OperationResponse>, IPaymentsRequestInitialBuilder
+{
+    private PaymentsRequestBuilder(Uri serverURI, HttpClient httpClient)
+        : base(serverURI, "payments", httpClient)
     {
-        public PaymentsRequestBuilder(Uri serverURI, HttpClient httpClient)
-            : base(serverURI, "payments", httpClient)
-        {
-        }
+    }
 
-        ///<Summary>
-        /// Builds request to <code>GET /accounts/{account}/payments</code>
-        /// <a href="https://www.stellar.org/developers/horizon/reference/payments-for-account.html">Effects for Account</a>
```

Comment:
The URL is no longer accessible. Same for others.

Reply:
You already fixed that in your change right? so we will fix it when we cherry pick everything to the main branch.

Reply:
I only fixed it for the URLs in the operation classes.

Reply:
Added them where is could find them removed the one I could not find.